how-to/configure-celery.rst

```diff
@@ -296,10 +300,14 @@ Set up local environment variables
 
 In ``.env-local`` add::
 
-    RABBITMQ_ERLANG_COOKIE=secret_cookie_value
-    BROKER_URL="amqp://guest:guest@rabbitmq:5672/"
+    DEFAULT_AMQP_BROKER_URL="amqp://guest:guest@rabbitmq:5672/"
 
-(Don't confuse the port ``5672`` of the RabbitMQ server with the port ``15672`` of its management interface.)
+..  note::
+
+    Applications using Aldryn Celery will require the environment variable ``DEFAULT_AMQP_BROKER_URL`` to be called
+    ``BROKER_URL``.
```

Comment:
"For legacy Aldryn Celery applications, name the environment variable ``BROKER_URL`` instead of ``DEFAULT_AMQP_BROKER_URL``."